src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -870,7 +875,8 @@ def _generate_preflight_query_conditions(
         else:
             queryset = queryset.order_by("date_added", "id")
 
-        assert limit is not None
+        # 100 is an arbitrary postgres limit added in other areas of the product
+        limit = limit if limit is not None else 100
```

Comment:
i would prefer to extract it to module level constant with `MAX_POSTGRES_LIMIT` or any other names.

Reply:
yeah good point.. wasn't done yet xd